pkg/controller/node/nodecontroller.go

```diff
@@ -343,6 +386,17 @@ func forcefullyDeletePod(c client.Interface, pod *api.Pod) {
 	}
 }
 
+// releaseCIDR does translating CIDR back to CIDR index and insert this index
+// back to availableCIDRs set
+func (nc *NodeController) releaseCIDR(freeCIDR string) {
+	CIDRsArray := strings.Split(freeCIDR, ".")
```

Comment:
If I'm not mistaken it's OK for /24, but it won't work for /26 or sth, as last token will have /26 at the end. It's not implemented now, but it's a straightforward change here, so let's make it future proof.